### PR TITLE
feat(DatePicker): Exporting DatePickerSelection

### PR DESCRIPTION
**Pull request by ogandose**
Exporting DatePickerSelection so that the DatePicker can be used without a Dropdown.

### CODE REVIEW

File: packages/axiom-components/src/index.js [hunk 1]
```diff
@@ -50,6 +50,7 @@ export { default as DataPicker } from './DataPicker/DataPicker';
 export { default as DataPickerDropdown } from './DataPicker/DataPickerDropdown';
 export { default as DataPickerMeta } from './DataPicker/DataPickerMeta';
 export { default as DatePicker } from './DatePicker/DatePicker';
+export { default as DatePickerSelection } from './DatePicker/DatePickerSelection';
```
Comment (tomru) [inline packages/axiom-components/src/index.js]: `DatePickerSelection` seems to be the right level to export for the use-case we discussed in the WG. It's not the best naming that I chose for it back then. Now that we export it, maybe it's time for a better name... Any good ideas?

File: packages/axiom-components/src/DatePicker/DatePickerSelection.js [hunk 2]
```diff
@@ -17,12 +16,46 @@ import './DatePicker.css';
 
 export default class DatePickerSelection extends Component {
   static propTypes = {
+    /**
+     * The date that should be shown to the user when the picker opens.
+     * When a selected date or range is given, this supersedes this
```
Comment (MariamBashir) [inline packages/axiom-components/src/DatePicker/DatePickerSelection.js]:
```suggestion
     * When a selected date or range is given, that supersedes this
```

File: packages/axiom-components/src/index.js (hunk 1 above)
Comment (lpoulter) [reply packages/axiom-components/src/index.js]: `DatePickerSelection` is ok. Other options I can think off;
*`DatePickerCalendar`
*`DatePickerSelectionCalendar`

Comment (tomru) [reply packages/axiom-components/src/index.js]: if you are fine maybe `DatePickerSelection` is just fine. My second choice would be `DatePickerCalendar` ....

Comment (tomru) [reply packages/axiom-components/src/index.js]: Let's leave it for now as `DatePickerSelection`...

**Pull request merged**